cg/clients/chanjo2/client.py

```diff
@@ -0,0 +1,37 @@
+"""Coverage analysis API for human clinical sequencing data."""
+
+import logging
+from typing import Any
+
+import requests
+
+from cg.clients.chanjo2.models import CoveragePostRequest, CoveragePostResponse
+from cg.clients.chanjo2.utils import handle_client_errors
+
+LOG = logging.getLogger(__name__)
```

Comment:
I guess all logging is done in the error handler now, so this can be removed? 😄

Reply:
Good catch! 

And thanks for the review, very useful suggestions 🙏